src/mlpack/methods/ann/layer/layer_types.hpp

```diff
@@ -259,8 +258,11 @@ using LayerTypes = boost::variant<
     NegativeLogLikelihood<arma::mat, arma::mat>*,
     Padding<arma::mat, arma::mat>*,
     PReLU<arma::mat, arma::mat>*,
+    Softmax<arma::mat, arma::mat>*,
+    TransposedConvolution<NaiveConvolution<ValidConvolution>,
+            NaiveConvolution<ValidConvolution>,
+            NaiveConvolution<ValidConvolution>, arma::mat, arma::mat>*,
     WeightNorm<arma::mat, arma::mat>*,
-    CELU<arma::mat, arma::mat>*,
     MoreTypes,
```

Comment:
Can you please restore the import re-arrangements made in this file? These are redundant.

Reply:
@ShikharJ: I rearranged the `LayerTypes` to maintain the alphabetical order as suggested by Mikhail. https://github.com/mlpack/mlpack/pull/2351#discussion_r423131788